aws-test/tests/aws_kinesis_firehose_delivery_stream/variables.tf

```diff
@@ -12,7 +12,7 @@ variable "aws_profile" {
 
 variable "aws_region" {
   type        = string
-  default     = "us-east-1"
+  default     = "ap-southeast-1"
```

Comment:
can you please revert this changes? The changes have already been merged into main https://github.com/turbot/steampipe-plugin-aws/pull/771/files#diff-799a5785764b494e2c76e1a05193b7873bf3c92ef08c1f0c3c851abf061429cdL9